src/_pytest/fixtures.py

```diff
@@ -256,15 +275,16 @@ def get_parametrized_fixture_keys(item: nodes.Item, scope: Scope) -> Iterator[_K
         for argname, param_index in sorted(cs.indices.items()):
             if cs._arg2scope[argname] != scope:
                 continue
+            param = SafeHashWrapper(cs.params.get(argname, param_index))
```

Comment:
A comment above this line explaining the purpose of `SafeHashWrapper` would be good.